platform/view/sdk/dig/sdk.go

```diff
@@ -121,7 +121,7 @@ func (p *SDK) Install() error {
 		p.C.Provide(operations.NewOperationSystem),
 		p.C.Provide(view3.NewResponseMarshaler, dig.As(new(view3.Marshaller))),
 		p.C.Provide(func(o *operations.Options, l operations.OperationsLogger) metrics2.Provider {
-			return operations.NewMetricsProvider(o.Metrics, l, false)
+			return operations.NewMetricsProvider(o.Metrics, l, true)
```

Comment:
why skipping the error?

Reply:
Because it happens to register the same metric multiple times. It should just ignore it and skip the double registration.